cartridge/shop/templates/admin/shop/product/change_list.html

```diff
@@ -0,0 +1,26 @@
+{% extends "admin/change_list.html" %}
+{% load shop_tags pages_tags i18n staticfiles %}
+
+{% block extrahead %}
+{{ block.super }}
+
+<script src="{% static "mezzanine/js/jquery-ui-1.8.14.custom.min.js" %}"></script>
+<script src="{% static "mezzanine/js/admin/page_tree.js" %}"></script>
+{% endblock %}
```

Comment:
Looks like all it does it goto the URL of the option tag that's selected, and page_tree.js has tons of other stuff related to managing a tree. We can probably just add a few lines of JS directly to handled the select change.